docs/Windows_Installation_Instructions.md

```diff
@@ -0,0 +1,176 @@
+    mms                   java -Djdk.tls.client.prot ...   Up      0.0.0.0:8080->8080/tcp
+    mms_elasticsearch_1   /usr/local/bin/docker-entr ...   Up      0.0.0.0:9200->9200/tcp, 0.0.0.0:9300->9300/tcp
+    mms_postgres_1        docker-entrypoint.sh postgres    Up      0.0.0.0:5432->5432/tcp
+    ```
+
+1. Or you can monitor what the server is doing by tailion and following the log for the `mms` container by:
+
+    ```bash
+    docker-compose logs -t -f mms
+    ```
+
+    which should show something like this:
+
+    ```bash
+    Attaching to mms
+    mms              | 2020-10-01T14:51:42.184914325Z
+    mms              | 2020-10-01T14:51:42.184964126Z   .   ____          _            __ _ _
+    mms              | 2020-10-01T14:51:42.184976826Z  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
+    mms              | 2020-10-01T14:51:42.184980226Z ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
+    mms              | 2020-10-01T14:51:42.184982726Z  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
+    mms              | 2020-10-01T14:51:42.184985026Z   '  |____| .__|_| |_|_| |_\__, | / / / /
+    mms              | 2020-10-01T14:51:42.185004626Z  =========|_|==============|___/=/_/_/_/
+    mms              | 2020-10-01T14:51:42.201916186Z  :: Spring Boot ::        (v2.2.6.RELEASE)
+    mms              | 2020-10-01T14:51:42.201936886Z
+    mms              | 2020-10-01T14:51:42.655378398Z 2020-10-01 14:51:42.646  INFO 1 --- [           main] o.o.sdvc.example.ExampleApplication      : Starting ExampleApplication on mms with PID 1 (/app.jar started by root in /mms)
+    mms              | 2020-10-01T14:51:42.659855214Z 2020-10-01 14:51:42.656  INFO 1 --- [           main] o.o.sdvc.example.ExampleApplication      : The following profiles are active: test
+    mms              | 2020-10-01T14:51:45.387823544Z 2020-10-01 14:51:45.386  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
+    mms              | 2020-10-01T14:51:45.713567912Z 2020-10-01 14:51:45.713  INFO 1 --- [           main] .s.d.r.c.RepositoryConfiguratio
+    ```
+
+## 6. Use MMS
+
+Refer to the MMS documentation, but to test if the service is running, you can go to: [`http://localhost:8080/healtcheck`](http://localhost:8080/healtcheck), which should show `healthy`.
+
+### Check out the REST API
+
+1. Go to [`http://localhost:8080/v3/swagger-ui.html`](http://localhost:8080/v3/swagger-ui.html) to see more info on the API.
+
+1. Expand the `POST /authentication` accordion and click on `Try it out`
+
+1. Set the Request body to be:
+    > The values below are based on the `application-test.properties` configuration file that was copied above, and the values for: `sdvc.admin.username` and `sdvc.admin.password`, if you changed that, make sure the values below match what you put in the application file.
+
+    ```json
```

Comment:
The new version allows you to authenticate within the modal now, so no need to copy-paste the token.  @dlamoris would you like me to update the instructions to reflect that?